README.md

```diff
@@ -1,11 +1,56 @@
-# redback-frontend-web-template
-This is a template repository used to centrally manage dependencies and shared code (other than what's in [redback-ui](https://github.com/Redback-Operations/redback-ui)) for all front-end web projects.
+# redback-fit-web
 
-The idea is that the project website/webapp repositories should fork off from this repo, and be updated to pull in any changes from here. This repository should NOT be directly used to create a website, and should only be forked from if there is a genuine need for a new website.
+Web application front-end and public-facing website for Redback Project 3 (Wearables for athletes).
```

Comment:
Looks like this PR should have been raised in redback-fit-web, not the template repository?

Reply:
I was demonstrating one of the team members on how to upload their code. How to fork, push their code and create pull requests. Will close the pull request now.